### PR TITLE
Amplitude - iOS v2 device mode update

### PR DESCRIPTION
**Pull request by ishachopra0401**
## Description of the change

Amplitude - iOS v2 device mode update

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: docs/destinations/analytics/amplitude.mdx [hunk 1]
```diff
@@ -222,11 +225,68 @@ Make sure that you enable <code class="inline-code">use IDFA as device id</code>
...
+        Follow these steps to add Amplitude to your iOS project:
+<ol>
+<li> Install <code class="inline-code">RudderAmplitude</code> by adding the following line to your <code class="inline-code">Podfile</code>.
+<span>
+
+```ruby
+pod 'RudderAmplitude', '~> 1.0.0'
+
+# for using IDFA as device id, location listening only
+pod 'Amplitude', '~> 8.8.0'
```
Comment (pallabmaiti) [inline docs/destinations/analytics/amplitude.mdx:243]: Don't need this line.

File: docs/destinations/analytics/amplitude.mdx [hunk 2]
```diff
@@ -222,11 +225,68 @@ Make sure that you enable <code class="inline-code">use IDFA as device id</code>
...
-</ul>
+</ol>
+      </TabPanel>
+        <TabPanel>
+          <div class="warningBlock">
+          This device mode integration is supported for Amplitude v8.8.0 and above.
+        </div>
+        Follow these steps to add Amplitude to your iOS project:
+<ol>
+<li> Install <code class="inline-code">RudderAmplitude</code> by adding the following line to your <code class="inline-code">Podfile</code>.
```
Comment (pallabmaiti) [inline docs/destinations/analytics/amplitude.mdx:236]:
```suggestion
<li> Install <code class="inline-code">RudderAmplitude</code> (available through [CocoaPods](https://cocoapods.org/)) by adding the following line to your <code class="inline-code">Podfile</code>.
```

**Review by pallabmaiti — CHANGES_REQUESTED** [review]

File: docs/destinations/analytics/amplitude.mdx [hunk 3]
```diff
@@ -150,16 +150,19 @@ This feature works only if the <strong>Track Referrer</strong> and <strong>Track
 
 ## Adding device mode integration
 
+Once you add Amplitude as a destination in the [RudderStack dashboard](https://app.rudderstack.com/), follow these steps to add it to your mobile project depending on your integration platform:
```
Comment (ameypv-rudder) [inline docs/destinations/analytics/amplitude.mdx:153]:
```suggestion
Once you add Amplitude as a destination in the [RudderStack dashboard](https://app.rudderstack.com/), follow these steps to add it to your project depending on your integration platform:
```

File: docs/destinations/analytics/amplitude.mdx [hunk 4]
```diff
@@ -150,16 +150,19 @@ This feature works only if the <strong>Track Referrer</strong> and <strong>Track
...
+
 <Tabs>
   <TabList>
     <Tab>iOS</Tab>
+    <Tab>iOS V2</Tab>
```
Comment (ameypv-rudder) [inline docs/destinations/analytics/amplitude.mdx:158]:
```suggestion
    <Tab>iOS v2</Tab>
```

File: docs/destinations/analytics/amplitude.mdx [hunk 5]
```diff
@@ -150,16 +150,19 @@ This feature works only if the <strong>Track Referrer</strong> and <strong>Track
...
     <Tab>Android</Tab>
     <Tab>React Native</Tab>
   </TabList>
     <TabPanels>
       <TabPanel>
         Follow these steps to add Amplitude to your iOS project:
-<ul>
+<ol>
 <li>In your <code class="inline-code">Podfile</code> and add the <code class="inline-code">Rudder-Amplitude</code> extension
```
Comment (ameypv-rudder) [inline docs/destinations/analytics/amplitude.mdx:166]:
```suggestion
<li>In your <code class="inline-code">Podfile</code>, add the <code class="inline-code">Rudder-Amplitude</code> extension, as shown:
```

File: docs/destinations/analytics/amplitude.mdx [hunk 6]
```diff
@@ -222,11 +225,65 @@ Make sure that you enable <code class="inline-code">use IDFA as device id</code>
...
-</ul>
+</ol>
+      </TabPanel>
+        <TabPanel>
+          <div class="warningBlock">
+          This device mode integration is supported for Amplitude v8.8.0 and above.
+        </div>
+        Follow these steps to add Amplitude to your iOS project:
+<ol>
+<li> Install <code class="inline-code">RudderAmplitude</code> (available through <a href="https://cocoapods.org/">CocoaPods</a>) by adding the following line to your <code class="inline-code">Podfile</code>.
```
Comment (ameypv-rudder) [inline docs/destinations/analytics/amplitude.mdx:236]:
```suggestion
<li> Install <code class="inline-code">RudderAmplitude</code> (available through <a href="https://cocoapods.org/">CocoaPods</a>) by adding the following line to your <code class="inline-code">Podfile</code>:
```

**Review by ameypv-rudder — APPROVED** [review]
Extremely minor suggestions, good to go once they are incorporated!

**Review by pallabmaiti — APPROVED** [review]

**Pull request merged**